packages/core/src/components/map-controls/map-controls.scss

```diff
@@ -34,16 +36,13 @@ $transition-duration: 300ms; // Sync with const transitionDuration in map-overla
   }
 }
 
-#controls {
-  align-items: start;
-  display: flex;
-  position: absolute;
-  right: calc(100% + 16px);
-  top: 16px;
-}
-
 #toggle-visibility-button {
   @include button_map();
+
+  left: auto;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/left `auto` is inderdaad de initial value. Prima dus, maar... waarom is het nodig? Ik zie niet waar er ergens `left` op zou komen?

Reply:
Na overleg tussen Ad en Pieter is de conclusie dat het niet nodig is